includes/class-wc-stripe-webhook-handler.php

```diff
@@ -1262,6 +1328,17 @@ public function process_deferred_webhook( $webhook_type, $additional_data ) {
 					}
 
 					$this->handle_deferred_payment_intent_succeeded( $order, $intent_id );
+					/**
+					 * Fires after a webhook has been processed, but before we respond to Stripe.
+					 * This allows for custom processing of the webhook after it has been processed.
+					 *
+					 * @since 9.8.0
+					 *
+					 * @param string $webhook_type The type of webhook that was processed.
+					 * @param object $notification The webhook data sent from Stripe.
+					 * @param WC_Order $order The order being processed by the webhook.
+					 */
+					do_action( 'wc_stripe_webhook_processed', (string) $webhook_type, $notification, $this->resolved_order );
```

Comment:
We should move this to after the `switch` to make it explicit that we expect it to be called for all async events... so we don't forget to call it when handling a new event type

Reply:
Good point. Moved it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4466/commits/c442208065845153f985e56f2ed5cf2c0605ff50